kymatio/scattering1d/frontend/base_frontend.py

```diff
@@ -581,6 +583,107 @@ def create_filters(self):
         # Check for absence of aliasing
         assert all((abs(psi1["xi"]) < 0.5/(2**psi1["j"])) for psi1 in psis_fr_f)
 
+    def scattering(self, x):
+        TimeFrequencyScatteringBase._check_runtime_args(self)
+        TimeFrequencyScatteringBase._check_input(self, x)
+
+        x_shape = self.backend.shape(x)
+        batch_shape, signal_shape = x_shape[:-1], x_shape[-1:]
+        x = self.backend.reshape_input(x, signal_shape)
+        U_0 = self.backend.pad(
+            x, pad_left=self.pad_left, pad_right=self.pad_right)
+
+        filters = [self.phi_f, self.psi1_f, self.psi2_f]
+        S_gen = joint_timefrequency_scattering(U_0, self.backend,
+            filters, self.oversampling, (self.average=='local'),
+            self.filters_fr, self.oversampling_fr, (self.average_fr=='local'))
+
+        # Zeroth order
+        path = next(S_gen)
+        if self.average == 'global':
+            path['coef'] = self.backend.average_global(path['coef'])
+        else:
+            res = self.log2_T if self.average else 0
+            path['coef'] = self.backend.unpad(
+                path['coef'], self.ind_start[res], self.ind_end[res])
+        path['coef'] = self.backend.reshape_output(
+            path['coef'], batch_shape, n_kept_dims=1)
+        S = [{**path, 'order': 0}]
+
+        # First and second order
+        for path in S_gen:
+            # "The phase of the integrand must be set to a constant. This
+            # freedom in setting the stationary phase to an arbitrary constant
+            # value suggests the existence of a gauge boson" — Glinsky
+            path['coef'] = self.backend.modulus(path['coef'])
+
+            # Temporal averaging and unpadding. Switch cases:
+            # 1. If averaging is global, no need for unpadding at all.
+            # 2. If averaging is local, averaging depends on order:
+            #     2a. at order 1, unpad at resolution log2_T
```

Comment:
because we've already averaged, before `frequency_scattering` and `modulus`.

https://github.com/kymatio/kymatio/blob/main/kymatio/scattering1d/core/timefrequency_scattering.py#L155

Those coefficients are:
```
    U_2(t, n1, n_fr) = ||x * psi_{n1}| * phi * psi_{n_fr}|
```
as you can see there is already a `phi`

Reply:
if you have an idea as to how to explain this in the comments, i'm all ears

Reply:
i see, thank you! maybe add some notations in the comment as a reminder? something like this:
`
# 2a. at order 1 S_gen yields Y_1_fr = S_1 * psi_{n_fr}, unpad at resolution log2_T
# 2b. at order 2, S_gen yields Y_2_fr = U_1 * psi_{n2} * psi_{n_fr} , average and unpad at resolution log2_T
`

Reply:
done in 66ddafe